src/main/java/com/amashchenko/maven/plugin/gitflow/AbstractGitFlowMojo.java

```diff
@@ -546,7 +549,12 @@ protected void initGitFlowConfig() throws MojoFailureException, CommandLineExcep
      */
     private void gitSetConfig(final String name, String value) throws MojoFailureException, CommandLineException {
         if (value == null || value.isEmpty()) {
-            value = "\"\"";
```

Comment:
before, there would be actually \"\" (with escape chars!) in the .gitconfig file. Only on Ubuntu, but not Windows.